app/assets/data/about_info.csv

```diff
@@ -0,0 +1,4 @@
+Name,Image,Email,Bio
+Luke Hunter James-Erickson,luke.png,theeluke@gmail.com,Backend Engineer~ D&D 5e DM
+Cecile Elliott,cecile.jpeg,cecile1981@gmail.com,UX/UI Designer~ D&D 5e DM
+Caleb Cyphers,caleb.png,caleb.j.cyphers@gmail.com,Frontend Engineer~ D&D 5e DM
```

Comment:
We can flesh out these bios more as needed, and because it's a CSV, that's all we'll have to update to update the About page